### PR TITLE
fix(github): remove gist- & follower-count for orgs

### PR DESCRIPTION
**Pull request by ghost**
Organizations can't have gists or followers, so this information can be omitted.

### CODE REVIEW

File: src/commands/main/github.ts [hunk 1]
```diff
@@ -72,7 +72,8 @@ export class GithubCommand extends SubCommand {
         try {
             const username = (await args.pickResult('string')).value as string;
             const { data } = await request('GET /users/{username}', { username });
-            const embed = new MessageEmbed()
+            if (data.type === 'User') {
+                const embed = new MessageEmbed()
                 .setTitle(data.login)
                 .setURL(data.html_url)
                 .setThumbnail(data.avatar_url)
```
Comment (jzeuzs) [inline src/commands/main/github.ts]: The indentation please.

File: src/commands/main/github.ts [hunk 2]
```diff
@@ -83,8 +84,20 @@ export class GithubCommand extends SubCommand {
                 .addField('Followers', String(data.followers), true)
                 .setFooter('Last updated at')
                 .setColor('RANDOM');
```
Comment (jzeuzs) [inline src/commands/main/github.ts]: Again, please indent.

File: src/commands/main/github.ts [hunk 3]
```diff
@@ -83,8 +84,20 @@ export class GithubCommand extends SubCommand {
...
+            } else {
+                const embed = new MessageEmbed()
+                .setTitle(data.login)
+                .setURL(data.html_url)
+                .setThumbnail(data.avatar_url)
+                .setTimestamp(data.updated_at)
+                .setDescription(data.bio ?? '')
+                .addField('Repositories', String(data.public_repos), true)
+                .setFooter('Last updated at')
+                .setColor('RANDOM');
```
Comment (jzeuzs) [inline src/commands/main/github.ts]: Indentation.

**Review by jzeuzs — CHANGES_REQUESTED** [review]

File: src/commands/main/github.ts (hunk 3 above)
Comment (ghost) [reply src/commands/main/github.ts]: My bad, should all be fixed now.

**Review by jzeuzs — APPROVED** [review]

**Pull request merged**